.travis.yml

```diff
@@ -23,10 +20,25 @@ script:
 - npm run lint
 - npm run test-travis
 
-deploy:
-- provider: script
-  skip_cleanup: true
-  script: npx semantic-release
-  on:
-    node: 22
-    branch: main
+jobs:
+  include:
+    - stage: Build-Test
+      language: node_js
+      node_js: 18
+    - node_js: 20
+    - node_js: 22
+
+    - name: Detect-Secrets
+      language: python
+      python: 3.12
+      install:
+        - pip install --upgrade "git+https://github.com/ibm/detect-secrets.git@master#egg=detect-secrets"
+      script:
+        - detect-secrets scan --update .secrets.baseline
+        - detect-secrets -v audit --report --fail-on-unaudited --fail-on-live --fail-on-audited-real .secrets.baseline
+
+    - stage: Semantic-Release
+      language: node_js
+      node_js: 22
+      script:
+        - npm run semantic-release
```

Comment:
We don't have a `semantic-release` NPM target in this repo, which is why we were using `npx` before. So we need to either update this script to use `npx` or add a script target in the `package.json` file like in [other cores](https://github.com/IBM/go-sdk-core/blob/main/package.json#L16). I'm happy either way

Reply:
I'm not the author, but nice catch. I don't think I would've been able to spot that... 😂

Reply:
I went ahead and added the script defn to package.json so that it's consistent with the other projects.